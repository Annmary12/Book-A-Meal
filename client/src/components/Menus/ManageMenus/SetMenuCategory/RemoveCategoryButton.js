import React from 'react';
import PropTypes from 'prop-types';
import classnames from 'classnames';
import cancelIcon from '../../../../../assets/img/cancel-icon.svg';
import styles from '../set-menu.scss';

const RemoveCategoryButton = ({
  handleRemoveCategory
}) => (
  <button
    className={classnames(styles.iconBtn, styles.right)}
    onClick={handleRemoveCategory}
  >
    <img src={cancelIcon} width="15" alt="cancel" />
  </button>
);

RemoveCategoryButton.propTypes = {
  handleRemoveCategory: PropTypes.func.isRequired
};

export default RemoveCategoryButton;
