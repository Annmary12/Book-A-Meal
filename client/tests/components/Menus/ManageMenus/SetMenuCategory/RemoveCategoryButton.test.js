import React from 'react';
import RemoveCategoryButton from '../../../../../src/components/Menus/ManageMenus/SetMenuCategory/RemoveCategoryButton';

describe('RemoveCategoryButton', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <RemoveCategoryButton
        handleRemoveCategory={() => {}}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});