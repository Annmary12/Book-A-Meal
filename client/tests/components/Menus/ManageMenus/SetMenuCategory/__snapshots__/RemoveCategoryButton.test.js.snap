// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveCategoryButton should match snapshot 1`] = `
<button
  className="iconBtn right"
  onClick={[Function]}
>
  <img
    alt="cancel"
    src="test-file-stub"
    width="15"
  />
</button>
`;
